docs/good-fuzz-target.md

```diff
@@ -0,0 +1,244 @@
+# What makes a good fuzz target
+
+This document explains the concept of a *fuzz target* and
+gives advice on how to make your fuzz targets really good.
+
+# Basics
+
+A *fuzz target* is a function that accepts an array of bytes and does something interesting with these bytes using the API under test.
```

Comment:
Do we want to make it even more generic? E.g. "a harness that passes the fuzzy input from the fuzzing engine to the target API(s)"? That should cover LPM-based targets + potentially other non-libFuzzer-based cases?

Reply:
I'd prefer to not have it too generic here. We can introduce more complicated examples later (in fact, we already do it in structure-aware-fuzzing)